internal/kuberesource/sets.go

```diff
@@ -638,3 +638,21 @@ func PatchServiceMeshAdminInterface(resources []any, port int32) []any {
 	}
 	return resources
 }
+
+// PatchCoordinatorMetrics enables Coordinator metrics on the specified port.
```

Comment:
Should this also add the `prometheus.io/scrape` annotation?

Reply:
As we only use this for E2E-tests, we don't need this annotation because we can discover the coordinator pods by name and port.